fake-cucumber/javascript/src/TestCase.ts

```diff
@@ -59,10 +60,13 @@ export default class TestCase implements ITestCase {
       testStepResults.push(testStepResult)
     }
 
+    const willBeRetried = retryable && getWorstTestStepResult(testStepResults).status === "FAILED"
```

Comment:
Can you use the enum constant here instead of "FAILED"? (I'm sure there are other places in the code we should clean this up).

Reply:
Done